src/index.js

```diff
@@ -1 +1,45 @@
 import './styles/style.css';
+import { getScores, postScore } from './modules/gamesapi.js';
+import showMessage from './modules/notifications.js';
+
+const renderScores = async () => {
+  const btnScores = document.getElementById('btn-scores');
+  btnScores.addEventListener('click', renderScores);
```

Comment:
New data is properly being submitted to the API and after the refresh button is clicked, new info is rendered as expected 🙌👌 Fantastic job!!